src/main/java/org/opentripplanner/routing/alternativelegs/AlternativeLegs.java

```diff
@@ -83,7 +84,8 @@ public static List<ScheduledTransitLeg> getAlternativeLegs(
     boolean searchBackward,
```

Comment:
I f you would be kind and rename this I would appreciate it. `searchBackward` is confusing - to me this indicate that we search from the destination towards the origin - we use search forward/reverse for this in other places of the code. I suggest using `departBefore`.